src/backend/catalog/pipeline_query.c

```diff
@@ -561,6 +564,25 @@ IsSWContView(RangeVar *name)
 	return sw;
 }
 
+/*
+ * IsTTLContView
+ */
+bool
+IsTTLContView(RangeVar *name)
+{
+	bool ttl = false;
+	HeapTuple tuple = GetPipelineQueryTuple(name);
+
+	if (HeapTupleIsValid(tuple))
+	{
+		Form_pipeline_query row = (Form_pipeline_query) GETSTRUCT(tuple);
+		ttl = AttributeNumberIsValid(row->sw_attno) || AttributeNumberIsValid(row->ttl_attno);
```

Comment:
For SW views, we should set the `ttl_attno` to the same as `sw_attno` an change this function to just check that attribute. It seems a little messy otherwise.